pkg/backend/httpstate/policypack.go

```diff
@@ -218,9 +218,12 @@ func installRequiredPolicy(finalDir string, tarball []byte) error {
 	fmt.Println()
 
 	// TODO[pulumi/pulumi#1307]: move to the language plugins so we don't have to hard code here.
-	err = npm.Install(finalDir, nil, nil)
+	err = npm.Install(finalDir, nil, os.Stderr)
```

Comment:
Just a UX question, as I know we faced this problem as part of `pulumi new`.  Does this cause warnings to be printed to the end user in the common case (i.e. is NPM warning about missing version number, license, etc)?  Would it make more sense to just buffer stderr (by passing something like a `bytes.Buffer` and then only print it if err is non nil)?

Reply:
It does not. We silence warnings with the appropriate npm flag.

Reply:
Also we're doing the same thing here now as we are in `new`, so the experience is very similar.

Reply:
:+1: